code/edgebridge/src/main/java/com/adobe/marketing/mobile/edge/bridge/EdgeBridgeMetricsBuilder.java

```diff
@@ -0,0 +1,66 @@
+/*
+  Copyright 2024 Adobe. All rights reserved.
+  This file is licensed to you under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License. You may obtain a copy
+  of the License at http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software distributed under
+  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+  OF ANY KIND, either express or implied. See the License for the specific language
+  governing permissions and limitations under the License.
+*/
+
+package com.adobe.marketing.mobile.edge.bridge;
+
+import static com.adobe.marketing.mobile.edge.bridge.EdgeBridgeConstants.LOG_TAG;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import com.adobe.marketing.mobile.services.AppState;
+import com.adobe.marketing.mobile.services.DeviceInforming;
+import com.adobe.marketing.mobile.services.Log;
+import com.adobe.marketing.mobile.services.ServiceProvider;
+import com.adobe.marketing.mobile.util.StringUtils;
+
+public class EdgeBridgeMetricsBuilder {
```

Comment:
- ⚠️ should be package protected
- This class is not really a builder, how about we call it EdgeBridgeProperties